### PR TITLE
Using WasmEdge C_API and Thread Proposal to Accelerate Compute-Intensive Workload

### PR DESCRIPTION
**Pull request by eee4017**

### CODE REVIEW

File: examples/capi/WasmEdge-Mandelbrot-Example/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+all: mandelbrot.so main
+
+mandelbrot.o: mandelbrot.c
+	clang-12 -O3 --no-standard-libraries --target=wasm32 -c -o $@ $^
```
Comment (hydai) [inline examples/capi/WasmEdge-Mandelbrot-Example/Makefile:5]: Do we need a specific version of clang?

File: examples/capi/WasmEdge-Mandelbrot-Example/main.cc [hunk 2]
```diff
@@ -0,0 +1,122 @@
+#include <wasmedge/wasmedge.h>
```
Comment (hydai) [inline examples/capi/WasmEdge-Mandelbrot-Example/main.cc:1]: Please follow the same coding style.

**Review by hydai — CHANGES_REQUESTED** [review]

**Comment by q82419** [comment]
Please rebase to the updated `proposal/threads`. Thanks.

**Comment by hydai** [comment]
Hi @eee4017 
Please raise another PR to the master directly. We have merged the threads proposal into master.

**Pull request closed without merging**